### PR TITLE
update tls docs with minimum supported version and cipher suites

### PR DESCRIPTION
**Pull request by ramaraochavali**
As discussed https://github.com/istio/istio/pull/27500#issuecomment-698942091 - adding docs for it.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ X] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by ramaraochavali** [comment]
@frankbu Sorry for the delay. Can you please take a look?

File: content/en/docs/concepts/security/index.md [hunk 1]
```diff
@@ -190,6 +190,15 @@ follows:
 1. After authorization, the server side Envoy forwards the traffic to the
    server service through local TCP connections.
 
+Istio configures TLSv1_2 as the minimum TLS version for both client and server with 
+following cipher suites:
```
Comment (ericvn) [inline content/en/docs/concepts/security/index.md:194]:
```suggestion
the following cipher suites:
```

File: content/en/docs/concepts/security/index.md [hunk 2]
```diff
@@ -190,6 +190,15 @@ follows:
...
+- CDHE-ECDSA-AES256-GCM-SHA384
```
Comment (ericvn) [inline content/en/docs/concepts/security/index.md:195]: I'm not sure if we should add all the to .spelling to get base lint or put them into something with isn't spell checked. Frank will have a better thought on that.

Comment (ramaraochavali) [reply content/en/docs/concepts/security/index.md:195]: @frankbu any thoughts on this?

File: content/en/docs/concepts/security/index.md [hunk 3]
```diff
@@ -190,6 +190,15 @@ follows:
...
+- ECDHE-RSA-AES256-GCM-SHA384
+- ECDHE-ECDSA-AES128-GCM-SHA256
+- ECDHE-RSA-AES128-GCM-SHA256
+- AES256-GCM-SHA384
+- AES128-GCM-SHA256
```
Comment (frankbu) [inline content/en/docs/concepts/security/index.md:200]: Let's escape these with back-ticks:
```suggestion
Istio configures `TLSv1_2` as the minimum TLS version for both client and server with 
following cipher suites:

- `CDHE-ECDSA-AES256-GCM-SHA384`
- `ECDHE-RSA-AES256-GCM-SHA384`
- `ECDHE-ECDSA-AES128-GCM-SHA256`
- `ECDHE-RSA-AES128-GCM-SHA256`
- `AES256-GCM-SHA384`
- `AES128-GCM-SHA256`
```

**Comment by ramaraochavali** [comment]
@ericvn @frankbu fixed. PTAL

File: content/en/docs/concepts/security/index.md [hunk 4]
```diff
@@ -190,6 +190,15 @@ follows:
 1. After authorization, the server side Envoy forwards the traffic to the
    server service through local TCP connections.
 
+Istio configures TLSv1_2 as the minimum TLS version for both client and server with
```
Comment (ericvn) [inline content/en/docs/concepts/security/index.md:193]:
```suggestion
Istio configures `TLSv1_2` as the minimum TLS version for both client and server with 
```

File: content/en/docs/concepts/security/index.md [hunk 5]
```diff
@@ -190,6 +190,15 @@ follows:
 1. After authorization, the server side Envoy forwards the traffic to the
    server service through local TCP connections.
 
+Istio configures TLSv1_2 as the minimum TLS version for both client and server with 
+the following cipher suites:
```
Comment (frankbu) [inline content/en/docs/concepts/security/index.md:194]: Fix lint error (trailing space on line 193). Also you need a blank line after 194, or the following list items will not display properly in [the preview](https://deploy-preview-8420--preliminary-istio.netlify.app/latest/docs/concepts/security/#mutual-tls-authentication).
```suggestion
Istio configures TLSv1_2 as the minimum TLS version for both client and server with
the following cipher suites:

```

**Comment by ramaraochavali** [comment]
@frankbu  lint is passing now. Please check

**Review by frankbu — APPROVED** [review]

**Pull request merged**